client/tests/nft_test.rs

```diff
@@ -18,7 +24,7 @@ async fn test_get_nft_data() {
 }
 
 #[tokio::test]
-async fn test_get_nft_collection_data() -> anyhow::Result<()> {
+async fn test_get_collection_data() -> anyhow::Result<()> {
```

Comment:
please revert to `test_get_nft_collection_data`

Reply:
Name of tested method: "get_collection_data()". So i suggest we keep this name or make ".test_nft_get_collection_data()"